timemachine/fe/utils.py

```diff
@@ -391,20 +391,20 @@ def get_romol_bonds(mol):
     return bond_list
 
 
-def get_romol_conf(mol) -> NDArray:
-    """Coordinates of mol's 0th conformer, in nanometers"""
-    conformer = mol.GetConformer(0)
+def get_romol_conf(mol, conf_id: int = 0) -> NDArray:
+    """Coordinates of the specified conformer, in nanometers"""
+    conformer = mol.GetConformer(conf_id)
     guest_conf = np.array(conformer.GetPositions(), dtype=np.float64)
-    return guest_conf / 10  # from angstroms to nm
+    return guest_conf / 10.0  # from angstroms to nm
 
 
-def set_romol_conf(mol, new_coords: NDArray):
+def set_romol_conf(mol, new_coords: NDArray, conf_id: int = 0):
     """Sets coordinates of mol's 0th conformer. Expects coords in nanometers and converts to angstrom"""
```

Comment:
Update the docstring

```suggestion
    """Sets coordinates of the specified conformer. Expects coords in nanometers and converts to angstrom"""
```

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1407/commits/aa7abb7eb54b8a37f60ff6a6e7b506dd0505a64a